src/babylon.scene.ts

```diff
@@ -2653,6 +2653,14 @@ module BABYLON {
             }
         }
 
+        /**
+         * Resets the last animation time frame.
+         * Useful to override when animations start running when loading a scene for the first time.
+         */
+        public resetLastAnimationTimeFrame(): void {
+            this._animationTimeLast = BABYLON.Tools.Now;

```

Comment:
Did you perhaps mean this? `BABYLON.Tools.Now` is a function.

```suggestion
            this._animationTimeLast = BABYLON.Tools.Now();
```